cmd/nsdns/utils.go

```diff
@@ -56,3 +63,35 @@ func GetKubernetesClientSet() (*kubernetes.Clientset, error) {
 
 	return nil, errors.New("failed to configure Kubernetes client")
 }
+
+func DomainManagerInformerFactory(dm *nsdns.DnsManager, clientset *kubernetes.Clientset) informers.SharedInformerFactory {
```

Comment:
Seems like a little bit of a pointless refactor. No real reason for this to move.